pkg/pgmodel/ingestor/copier.go

```diff
@@ -446,8 +446,6 @@ func insertSeries(ctx context.Context, conn pgxconn.PgxConn, reqs ...copyRequest
 			registerDuplicates(numRowsExpected - insertedRows)
 		}
 	}
-	metrics.IngestorInsertablesIngested.With(prometheus.Labels{"type": "metric", "kind": "sample"}).Add(float64(totalSamples))
```

Comment:
I'd rather update metrics in the same place where ingest happens than via callbacks ( unless there is a special reason). Mostly for keeping things simple